### PR TITLE
Docs: Setting up Initium platform in AKS Cluster

### LINKED ISSUE #165: Test the platform on AKS and write a aks-quick-start

### PR DESCRIPTION
**Pull request by jeevanions**
Provides a quick start guide to create an AKS cluster in Azure and set up the Initium platform. It also provides instructions for deploying a node-based demo app.

Closes https://github.com/nearform/initium-platform/issues/165

### CODE REVIEW

File: quick-start-aks.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+``` bash
+az account -s <Subscription name or Id>
+```
+## Create AKS Cluster for Dev/Test Purpose
+
+From the command line run below command to create a AKS cluster. Note that you need to create a new resource group to host the AKS resource.
+
+``` bash
+AKS_RESOURCE_GROUP="<Your Resource Group Name>"
```
Comment (LucaLanziani) [inline quick-start-aks.md:30]: Can we move this to a separate code block and I'm not sure this is going to work if the user doesn't export the variables.

File: quick-start-aks.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+AKS_CLUSTER="initium-test-aks-cluster" # Set the name of the cluster as you require
```
Comment (LucaLanziani) [inline quick-start-aks.md:31]: Similarly, this should be `export AKS_CLUSTER....`

File: quick-start-aks.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+# Create Log Analytics Workspace
+AKS_MONITORING_LOG_ANALYTICS_WORKSPACE_ID=$(az monitor log-analytics workspace create \
```
Comment (LucaLanziani) [inline quick-start-aks.md:34]: same here

File: quick-start-aks.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+
+Login using CLI as from above from the root of the `initium-platform` repo 
+
+>Note that at this point you would have a AKS Cluster ready to use.
+
+### Check Cluster access
+
+``` bash
+
+AKS_RESOURCE_GROUP="<Update your resource group>"
```
Comment (LucaLanziani) [inline quick-start-aks.md:92]: Similar to the one above, let's move this to a separate code block so once I've set this I can copy and paste the rest without having to modify anything.

File: quick-start-aks.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+AKS_CLUSTER="initium-test-aks-cluster"
```
Comment (LucaLanziani) [inline quick-start-aks.md:93]: I believe we need an `export` here

File: quick-start-aks.md (hunk 1 above)
Comment (jeevanearform) [reply quick-start-aks.md]: I tested this and it worked without the export. But anyway added the export now.

File: quick-start-aks.md (hunk 2 above)
Comment (jeevanearform) [reply quick-start-aks.md]: Added

File: quick-start-aks.md (hunk 3 above)
Comment (jeevanearform) [reply quick-start-aks.md]: Added

File: quick-start-aks.md (hunk 5 above)
Comment (jeevanearform) [reply quick-start-aks.md]: Added

File: quick-start-aks.md (hunk 4 above)
Comment (jeevanearform) [reply quick-start-aks.md]: Added

**Review by Pehesi97 — CHANGES_REQUESTED** [review]
Sorry for these many requested changes. Overall the guide is great.

**Pull request merged**